spec/models/manageiq/providers/openstack/cloud_manager/event_catcher/event_catcher_runner_spec.rb

```diff
@@ -0,0 +1,30 @@
+describe ManageIQ::Providers::Openstack::CloudManager::EventCatcher::Runner do
```

Comment:
I think this spec is testing functionality that should be tested for the base class, namely that filtered events wont be passed to `proccess_event`. I would defer testing this because a) you are going to refactor this more and b) its really [trivial](https://github.com/durandom/manageiq/blob/c6fd476bbbde7903277e7b87200248a219f661ab/app/models/manageiq/providers/base_manager/event_catcher/runner.rb#L130-L141)

the specs in here should rather test when an event is filtered